### DISCUSSION TITLE
Remove presentation data from Databasebuilder return values

### DISCUSSION THREAD

**Discussion by Migaroez** [umbraco/Umbraco-CMS#14734] [Features and ideas]
The `DatabaseBuilder` inside `Umbraco.Cms.Infrastructure.Migrations.Install` **sometimes** returns html in its return value.
Since html is part of the presentation layer, it should not be there.

Would be a good idea to remove the p tags and fix the frontend layout if this breaks things.

**Comment by Migaroez** [comment]
After talking to @nielslyngsoe, it should probably be changed to returning (readable)localization keys so the frontend can update/manage the messages themselves and can be overwritten like anything else.
I state readable because even though the API is only consumed by our own frontend, it still shows up in the logs and I think it's a good habit to reduce unclear/cryptic log messages/return values.

@kjac  would live your take on this.

**Reply by kjac** [reply]
Makes perfect sense to me.

In any case, this will be changed in V14, where localization no longer becomes a concern on the server (with the exception of sending system emails).
